tests/standalone/run_all.py

```diff
@@ -264,6 +264,15 @@
     "MFCM140U.DLL",
 )
 
+# finds the special comment in each test module and returns the import module needed for each test
+# special comments are the the following format:
+# "# nuitka-skip-test-unless: module"
+def find_required_module(filename):
+    with open(filename,'r') as f:
+        for line in f:
+            if '# nuitka-skip-test-unless: ' in line:
```

Comment:
You made a mistake here, you are checking with 'in' for mere presence. I would argue, that is better than 'startswith', but then you use hardcoded slicing below, which presumes that it's at the beginning of the line.